docs/non-normative/status-metrics.md

```diff
@@ -0,0 +1,95 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Status Metrics
+aliases: [status-metrics]
+--->
+
+# State Metrics
+
+**Status**: [Development][DocumentStatus]
+
+<!-- toc -->
+
+- [Definition](#definition)
+- [Design](#design)
+  - [Naming](#naming)
+  - [Instrument](#instrument)
+  - [Why not Resource Attributes?](#why-not-resource-attributes)
+
+<!-- tocstop -->
+
+## Definition
+
+For the purposes of this document, a "status metric" is a metric used to
+represent something being in a particular state at a given time from a closed
+set of distinct possible state values.
+
+## Design
+
+To represent a metric of this nature, the design will look like the following:
+
+```yaml
+id: metric.example.status
+type: metric
+metric_name: example.status
+stability: development
+brief: "The current state."
+note: |
+  A timeseries is produced for every possible value of `example.state`. The
+  value of this metric is 1 for a state if it is currently in said state, and
+  is 0 for all other states.
+instrument: updowncounter
+unit: "1"
+attributes:
+- ref: example.state
+```
+
+### Naming
```

Comment:
could you summarize it to one sentence and add a point (with link) to https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/naming.md#metrics ?